Tests/Validator/InlineValidatorTest.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\Tests\Validator;
+
+use Sonata\CoreBundle\Tests\Fixtures\Bundle\Validator\FooValidatorService;
+use Sonata\CoreBundle\Validator\ErrorElement;
+use Sonata\CoreBundle\Validator\InlineValidator;
+
+/**
+ * Test for InlineValidator.
+ *
+ * @author Ahmet Akbana <ahmetakbana@gmail.com>
+ */
+final class InlineValidatorTest extends \PHPUnit_Framework_TestCase
+{
+    private $container;
+    private $constraintValidatorFactory;
+    private $context;
+
+    public function setUp()
+    {
+        $this->container = $this->getMock('Symfony\Component\DependencyInjection\ContainerInterface');
+        $this->constraintValidatorFactory = $this->getMock(
+            'Symfony\Component\Validator\ConstraintValidatorFactoryInterface'
+        );
+        $this->context = $this->getMock(
+            interface_exists('Symfony\Component\Validator\Context\ExecutionContextInterface') ?
+                'Symfony\Component\Validator\Context\ExecutionContextInterface' :
+                'Symfony\Component\Validator\ExecutionContextInterface'
+        );
+    }
+
+    public function testGetErrorElement()
+    {
+        $inlineValidator = new InlineValidator($this->container, $this->constraintValidatorFactory);
+
+        $reflectorClass = new \ReflectionClass($inlineValidator);
+        $reflectedProperty = $reflectorClass->getProperty('context');
+        $reflectedProperty->setAccessible(true);
+        $reflectedProperty->setValue($inlineValidator, $this->context);
```

Comment:
:dizzy_face: ah
Yes of course. I didn't really analyze the parent class, just didn't see any setter for the property.